pkg/cmd/pulumi/new_test.go

```diff
@@ -807,6 +807,10 @@ func removeStack(t *testing.T, name string) {
 }
 
 func skipIfShort(t *testing.T) {
+	_, ok := os.LookupEnv("PULUMI_ACCESS_TOKEN")
+	if !ok {
+		t.Skipf("Skipping: PULUMI_ACCESS_TOKEN is not set")
+	}
```

Comment:
It's not immediately clear to me why this is right to do in `skipIfShort` - how is this check related to whether the tests are running in "short" mode?

Perhaps just the name of this function needs to change to `skipIfShortOrNoPulumiAccessToken`?

Reply:
It's probably not, I was really just trying to take advantage of the existing function being available.

I've updated the name as suggested